package.json

```diff
@@ -33,6 +33,7 @@
     "@types/masknet__global-types": "workspace:*",
     "@types/react": "^17.0.39",
     "@types/react-dom": "^17.0.11",
+    "graphql-request": "^4.0.0",
```

Comment:
Why you need to install it as a top-level package?

Reply:
yes, removed